mergify_engine/engine/actions_runner.py

```diff
@@ -107,38 +107,70 @@ async def gen_summary(
     summary = ""
     summary += await get_already_merged_summary(ctxt, match)
 
+    matching_rules_to_display = match.matching_rules[:]
+    not_applicable_base_changeable_attributes_rules_to_display = []
+    for rule in match.matching_rules:
+        if rule in match.not_applicable_base_changeable_attributes_rules:
+            conditions = rule.conditions.copy()
+            for condition in conditions.walk():
+                attr = condition.get_attribute_name()
+                if attr in match.BASE_CHANGEABLE_ATTRIBUTES and not condition.match:
+                    matching_rules_to_display.remove(rule)
+                    not_applicable_base_changeable_attributes_rules_to_display.append(
+                        rule
+                    )
```

Comment:
I don't see why we check BASE_CHANGEABLE_ATTRIBUTES, I think this should be enough:

```suggestion
    for rule in match.matching_rules:
        if rule in match.not_applicable_base_changeable_attributes_rules:
                matching_rules_to_display.remove(rule)
                not_applicable_base_changeable_attributes_rules_to_display.append(
                    rule
                )
```